src/main/java/com/solr/clientwrapper/rest/SolrDocParseResource.java

```diff
@@ -0,0 +1,156 @@
+package com.solr.clientwrapper.rest;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Set;
+import java.net.URL;
+import org.json.JSONArray;
+import org.json.JSONObject;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.mysql.cj.xdevapi.JsonArray;
+
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+import java.net.HttpURLConnection;
+
+
+
+
+@RestController
+@RequestMapping("/file")
+public class SolrDocParseResource {
+
+	@RequestMapping(value = "/upload", method = RequestMethod.POST, consumes = "application/json", produces = "application/json")
+	public String sendjson(@RequestBody String payload) throws Exception {
+
+		// String urlString = "http://localhost:8983/solr/user";
+		String url = "http://localhost:8983/solr/docparse/schema/fields?wt=json";
+		// SolrClient Solr = new HttpSolrClient.Builder(urlString).build();
+		// SolrInputDocument doc = new SolrInputDocument();
+
+		URL obj = new URL(url);
+		HttpURLConnection con = (HttpURLConnection) obj.openConnection();
+		// optional default is GET
+		con.setRequestMethod("GET");
+		// add request header
+		 con.setRequestProperty("User-Agent", "Mozilla/5.0");
+		BufferedReader in = new BufferedReader(new InputStreamReader(con.getInputStream()));
+		String inputLine;
+		StringBuffer response = new StringBuffer();
+		while ((inputLine = in.readLine()) != null) {
+			response.append(inputLine);
+		}
+		in.close();
```

Comment:
SolrClient solrClient=new HttpSolrClient.Builder("http://localhost:8983/solr/S3Collection").build();

        SchemaRequest schemaRequest = new SchemaRequest();
        SchemaResponse schemaResponse = schemaRequest.process(solrClient);

        System.out.println(schemaResponse.getSchemaRepresentation().getFields());

        return schemaResponse.getSchemaRepresentation().getFields().toString();

Reply:
This is the Spring Data Solr Equivalent